lib/modules/accountSwitcher.js

```diff
@@ -434,7 +463,36 @@ function _notifySwitchedAccountElsewhere() {
 		message,
 	});
 }
-
+async function autoSwitched() {
+	return Date.now() - await lastAutoSwitched.get() < 6000;
```

Comment:
A more reliable way to notify about this may be to add a token to `sessionStorage`. Then, when loading a page, it can be checked whether the token exist and if it does, show the notification; if not, run the code which looks for another user.

The source of the switch can also be stored, i.e. `post` or `comment`, so that it won't be necessary to look for the author in the posts when creating the notification.